src/01-functions.js

```diff
@@ -112,7 +124,13 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if (typeof a === typeof b) {
```

Comment:
Is the conditional here necessary? Try refactoring without it. (Hint: your solution may be a single return statement)